Fomichev/HashingApp/HashingApp/MainForm.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Text;
+using System.Windows.Forms;
+using DataStructs;
+using DataStructs.Types;
+using Implements.Managers;
+using Implements.Services;
+using Implements.Validators;
+using ValidationException = DataStructs.Exceptions.ValidationException;
+
+namespace HashingApp
+{
+    public partial class MainForm : Form
+    {
+        public MainForm() => InitializeComponent();
```

Comment:
Just curious - is Visual Studio now creating projects like this, with lambda constructors everywhere?
Or did you write it?

Reply:
I wrote it)